tests/integration/cmd_url_test.go

```diff
@@ -38,7 +37,8 @@ var _ = Describe("odo url command tests", func() {
 			helper.CmdShouldPass("odo", "url", "create", url1, "--port", "8080", "--context", commonVar.Context)
 			stdout = helper.CmdShouldPass("odo", "url", "list", "--context", commonVar.Context)
 			helper.MatchAllInOutput(stdout, []string{url1, "Not Pushed", url1, "odo push"})
-			helper.DontMatchAllInOutput(stdout, []string{"://"})
+			// Looks like the next line is catching a bug, commenting it while I confirm if it is a real bug
+			//			helper.DontMatchAllInOutput(stdout, []string{"://"})
```

Comment:
You can confirm the url list output by manually trying the steps. I don't think commenting out the test would help get it resolved.

Reply:
I agree that commenting the code does not resolve it. I ran the commands manually and I get the output that causes the failure. I posted a question https://coreos.slack.com/archives/G01BY1NC88L/p1608143410002500 asking to confirm if this is a bug.

Reply:
comment removed and passes, so the problem is specific to PSI